.all-contributorsrc

```diff
@@ -218,6 +218,13 @@
         "code",
         "test"
       ]
+    },
+    {
+      "login": "citycide",
+      "name": "Bo Lingen",
+      "avatar_url": "https://avatars.githubusercontent.com/u/16605186?v=3",
+      "profile": "https://github.com/citycide",
+      "contributions": []
```

Comment:
This should probably have `"doc"` in the array. You can add that manually then run `npm run generate-contributors`.